zeppelin-web/src/app/jobmanager/job/job.html

```diff
@@ -27,7 +27,8 @@
         class="job-control-btn" tooltip-placement="left"
         uib-tooltip-html="!$ctrl.isRunning() ? 'Start All Paragraphs' : 'Stop All Paragraphs'"
         ng-click="!$ctrl.isRunning() ? $ctrl.runJob() : $ctrl.stopJob()"
-        ng-class="!$ctrl.isRunning() ? 'icon-control-play' : 'icon-control-pause'">
+        ng-class="!$ctrl.isRunning() ? 'icon-control-play' : 'icon-control-pause'"
+        ng-style="{'color': $ctrl.isRunning() ? '#CD5C5C' : '#3071A9'}">
```

Comment:
instead of just adding `ng-style`, it would be nice to add new func in `job-status.js` and then use it

like 

https://github.com/SDRLurker/zeppelin/blob/d7325c1cda5c671aa5e5337bcd97af2b29be340e/zeppelin-web/src/app/jobmanager/job-status.js#L40-#L54